tests/sentry/monitors/endpoints/test_monitor_ingest_checkin_details.py

```diff
@@ -192,7 +192,7 @@ def test_failing(self):
             assert monitor_environment.next_checkin > checkin.date_added
             assert monitor_environment.next_checkin_latest > checkin.date_added
             assert monitor_environment.status == MonitorStatus.ERROR
-            assert monitor_environment.last_checkin > checkin.date_added
+            assert monitor_environment.last_checkin == checkin.date_added
```

Comment:
This is reflecting the change to `last_checkin` being the time of the check-ins creation, NOT the last time it was updated.